client/my-sites/woocommerce/woop/landing-page.tsx

```diff
@@ -4,15 +4,11 @@ import styled from '@emotion/styled';
 import { useRef } from '@wordpress/element';
 import { useI18n } from '@wordpress/react-i18n';
 import page from 'page';
-import Image01 from 'calypso/assets/images/woocommerce/woop-cta-image01.jpeg';
-import Image02 from 'calypso/assets/images/woocommerce/woop-cta-image02.jpeg';
-import Image03 from 'calypso/assets/images/woocommerce/woop-cta-image03.jpeg';
-import Image04 from 'calypso/assets/images/woocommerce/woop-cta-image04.jpeg';
```

Comment:
Should we delete those images since they're not being used anymore?

Reply:
That's on the list as a non-blocking change I believe: https://github.com/Automattic/wp-calypso/pull/60697#issuecomment-1026528667 Do you think it's a blocker?

Reply:
Ah, gotcha! In that case, no :)